app/components/Header.tsx

```diff
@@ -0,0 +1,30 @@
+import Kopfzeile from "~/components/Kopfzeile.static";
+import Logo from "~/components/Logo.static";
+import Navigation from "~/components/Navigation.static";
+import UserProfile from "~/components/UserProfile.static";
+
+export default function Header({
+  userIsLoggedIn,
+}: {
+  userIsLoggedIn: boolean;
+}) {
+  return (
+    <header>
+      <Kopfzeile />
+      {userIsLoggedIn && (
+        <div className="kern-container">
+          <div className="gap-kern-space-small flex flex-col">
```

Comment:
This needs a bit more space to the top:
```suggestion
          <div className="gap-kern-space-small pt-kern-space-large flex flex-col">
```